layouts/partials/head.html

```diff
@@ -80,6 +80,8 @@
 {{ end }}
 
 <script src="/js/jquery-3.6.0.min.js" intregrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
+<!--script src="https://unpkg.com/split.js/dist/split.min.js"></script-->
+<script src="/js/split-1.6.0.js"></script>
```

Comment:
We could set an `integrity` value too, even though it's local. Helps track when we made changes.